content/product/product_management/index.md

```diff
@@ -20,15 +20,20 @@ Sourcegraph has a top-level [strategy page](../../company/strategy/index.md) tha
 
 In addition to this shared content, each team has a set of sources of truth (that do not overlap) that they should update regularly:
 
-#### Per-area strategy pages
+#### OKR and roadmap tracking
 
-We also use [per-area strategy pages](../../company/strategy/index.md#per-area-strategy-pages) to communicate where each of our product teams is headed, for the areas of the product that they work on. Newly created teams should begin documenting the scope of their team as well as their mission, vision, principles, and more by creating one of these pages as part of the process of forming. Having a page like this will help you ensure your own framework for prioritization is clear, and can also be a resource for those outside your team who want to understand why you are working on what you are.
+We have a [high-level OKR and roadmap tracker](https://github.com/orgs/sourcegraph/projects/214/views/1) (internal only) in GitHub. Each month you should work with your engineering manager to update this content, which is the source of truth for goal and roadmap tracking.
 
-These are updated monthly by each team, but wouldn't normally change too much each month. They are updated via a pull request, that anyone on the team can contribute to, and that can be shared with stakeholders as a summary of the change.
+After updating this content, you should ensure it is reflected in the manually created slide decks which rely on this information:
+
+- The [PMM roadmap deck](https://docs.google.com/presentation/d/1o3R8WUIhzzRz0x5laTwVcizOzVWrMBe5MCAz74H45Ss/edit#slide=id.gf131fe1596_2_7) (internal only) deck which contains upcoming and recently launched important customer-facing features from the OKR and roadmap tracker.
+- The [Key accounts deck](https://docs.google.com/presentation/d/18hw513mX3ssA9isQdGdnmrt5-BIN4TRMpjkSaodTEuw/edit#slide=id.gf03521df2b_0_0) (internal only) contains status information on key accounts.
```

Comment:
Similar question - I don't think we answered who owns this. @christinaforney or @nicksnyder do you own copying the data from the GitHub dashboard into the key accounts deck on some cadence or do you expect product managers to do it when they make updates to the dashboard?

Reply:
Maybe another related idea, could the OKR deck and key accounts deck be combined to reduce the number places to manually copy info to?

Reply:
I own this (with @kyfli 's help). The key accounts deck contains the 5 slides about our key customers and is updated primarily by the CEs and AEs. The OKR deck didn't contain exact copied info, but pulls up the most important info to surface and track. I think they serve different purposes. 

I think the thing that I haven't solved for yet is what information is tracked in the GitHub board as the source of truth (vs. the key accounts deck). I'll circle around with Aimee and Gregg to see if we need to make any changes. 

TLDR; I own this, and the copying burden for this shouldn't fall on the PMs

Reply:
Thanks @christinaforney !

```suggestion
- The [Key accounts deck](https://docs.google.com/presentation/d/18hw513mX3ssA9isQdGdnmrt5-BIN4TRMpjkSaodTEuw/edit#slide=id.gf03521df2b_0_0) (internal only) contains status information on key accounts. Christina and Kylie own making this update for the product team.
```